signUrl/README.md

```diff
@@ -0,0 +1,56 @@
+# Google Cloud Signing URL Sample
```

Comment:
other samples seem to be following Pascal case (i.e. `SignUrl`), is it applicable here?

Reply:
It's just the folder name, where it always starts with small letter. The rest is following the same convension.

Reply:
actually I think this should be in `cdn/signedUrl` directory, not `signedUrl`.